zstash/extract.py

```diff
@@ -12,7 +12,7 @@
 import tarfile
 import traceback
 from datetime import datetime
-from typing import List
+from typing import DefaultDict, List, Optional, Tuple
 
 from . import parallel
 from .hpss import hpss_get
```

Comment:
For some reason I can't comment on chunks of code that are collapsed in the diff viewer.

Anyways, need to remove lines 28, 29, and 174 like the other files (related to `con` and `cur`)